### PR TITLE
Couple notes on React

### PR DESCRIPTION
**Pull request by justin808**
@jbhatab @alexfedoseev @josiasds @robwise OK?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,46 @@ Here's a few notes on top of that:
...
+  
+  static defaultProps = {
+    initialCount: 0,
+  };
+  
+  constructor(props) {
+    super(props);
+    this.state = {count: props.initialCount};
+    
+    _.bindAll("tick");
```
Comment (alex35mil) [inline README.md]:
``` js
_.bindAll(this, 'tick');
          ^     ^
```

File: README.md [hunk 2]
```diff
@@ -12,6 +12,46 @@ Here's a few notes on top of that:
...
+We mostly folow the official [FB docs on ES6 for React](https://facebook.github.io/react/docs/reusable-components.html#es6-classes). We initialize state in the constructor. 
+
+Notable differences:
+
+1. ES7 static syntax for the propTypes and defaultProps
+2. Use lodash `_.bindAll` for callback bindings.
+
+For example:
+
+```javascript
```
Comment (alex35mil) [inline README.md:25]:
``` js
import React, { PropTypes } from 'react';
```

File: README.md [hunk 3]
```diff
@@ -12,6 +12,46 @@ Here's a few notes on top of that:
...
+import _ from 'lodash';
+export class Counter extends React.Component {
+  static propTypes = {
+    initialCount: React.PropTypes.number,
```
Comment (alex35mil) [inline README.md]:
``` js
initialCount: PropTypes.number,
```

File: README.md [hunk 4]
```diff
@@ -12,6 +12,46 @@ Here's a few notes on top of that:
...
+Notable differences:
+
+1. ES7 static syntax for the propTypes and defaultProps
+2. Use lodash `_.bindAll` for callback bindings.
+
+For example:
+
+```javascript
+import _ from 'lodash';
+export class Counter extends React.Component {
```
Comment (alex35mil) [inline README.md]:
``` js
export default ...
```

File: README.md [hunk 5]
```diff
@@ -12,6 +12,46 @@ Here's a few notes on top of that:
...
+    initialCount: React.PropTypes.number,
+  };
+  
+  static defaultProps = {
+    initialCount: 0,
+  };
+  
+  constructor(props) {
+    super(props);
+    this.state = {count: props.initialCount};
```
Comment (alex35mil) [inline README.md]:
``` js
{ count: props.initialCount }
 ^                         ^
```

**Comment by alex35mil** [comment]
All good! :+1: A few stylistic suggestions.

File: README.md [hunk 6]
```diff
@@ -12,6 +12,47 @@ Here's a few notes on top of that:
...
+  constructor(props) {
+    super(props);
+    this.state = { count: props.initialCount };
+    
+    _.bindAll(this, 'tick');
+  }
+  tick() {
+    this.setState({ count: this.state.count + 1 });
+  }
+  render() {
```
Comment (robwise) [inline README.md:46]: @justin808 @alexfedoseev we don't use blank lines in between methods?

Comment (alex35mil) [inline README.md:46]: Use actually, good catch :+1:

**Pull request merged**